src/types.d.ts

```diff
@@ -0,0 +1,5 @@
+export interface AtomValue {
+  content: string;
```

Comment:
Can `content` be a generic value, or `any`? This may be a char for an `InsertChar` atom, or an increment for an `Add` atom.